src/main/scala/com/workday/warp/persistence/mysql/HasWarpSlickDsl.scala

```diff
@@ -29,7 +29,26 @@ trait HasWarpSlickDsl {
       }
       expression.apply(str, pattern)
     }
-
+}
+  /**
+    * dsl class for math related operation
+    */
+  object numberExtension {
```

Comment:
Oops missed this. Can we capitalize `N` in number. For class/object/trait declarations we usually use capitalized camel case